lib/cdo/metrics_helper.rb

```diff
@@ -2,7 +2,7 @@
 
 module Metrics
   DEVINTERNAL_DB = CDO.devinternal_db_writer ?
-    sequel_connect(CDO.devinternal_db_writer, CDO.devinternal_db_writer) : nil
+    sequel_connect(CDO.devinternal_db_writer, CDO.devinternal_db_writer, -1) : nil
```

Comment:
This -1 seems mysterious - can we add a comment about it?

Reply:
Good call! Adding . . .